app/views/layouts/_base.html.erb

```diff
@@ -66,40 +66,7 @@
 <% content_for :body_end do %>
   <%= javascript_include_tag 'application' %>
 
-  <% if homepage_experiment_id %>
-    <%= javascript_include_tag 'https://cdn3.userzoom.com/uz.js?cuid=C85E515772DAE311BEDA0022196C4538' %>
-    <script>//<![CDATA[
-    var experimentMatches = document.cookie.match(/<%= ENV["HOMEPAGE_EXPERIMENT_ID"] %>:([0-1])/),
-      experimentId = 0;
-
-    if (experimentMatches !== null && experimentMatches[1]) {
-      experimentId = parseInt(experimentMatches[1]);
-    }
-
-    _uzactions = window._uzactions || [];
-
-    if (experimentId === 0) {
-      _uzactions.push(['_setID', '3F71AF71102FE6118CAD0022196C2B88']);
-      _uzactions.push(['_setSID', '3E71AF71102FE6118CAD0022196C2B88']);
-    } else {
-      _uzactions.push(['_setID', '6AF15392153AE611A8C60022196C2B88']);
-      _uzactions.push(['_setSID', '69F15392153AE611A8C60022196C2B88']);
-    }
-
-    _uzactions.push(['_start']);
-
-    (function() {
-      var uz = document.createElement('script');
-      uz.type = 'text/javascript';
-      uz.async = true;
-      uz.charset = 'utf-8';
-      uz.src = 'https://cdn4.userzoom.com/trueintent/js/uz_til.js?cuid=C85E515772DAE311BEDA0022196C4538';
-      var s = document.getElementsByTagName('script')[0];
-      s.parentNode.insertBefore(uz, s);
-    })();
-    //]]>
-    </script>
-  <% end %>
+  <%= render 'testing/user_zoom' %>
```

Comment:
Tiny thing but shall we call this directory `research` instead of testing? Just feels a bit less ambiguous.